buildSrc/src/main/rocker/com/uber/okbuck/template/android/KeystoreRule.rocker.raw

```diff
@@ -1,14 +1,12 @@
 @args (
-String name,
 String storeFile,
 String storePassword,
 String keyAlias,
 String keyPassword
 )
-okbuck_keystore(
-  name = '@name',
-  storeFile = '@storeFile',
-  storePassword = '@storePassword',
-  keyAlias = '@keyAlias',
-  keyPassword = '@keyPassword'
-)
+@com.uber.okbuck.template.base.BuckRule.template() -> {
```

Comment:
technically you don't need to inherit from the template, instead you may just use `@(ruleType)` instead of `okbuck_keystore` 

this may be cleaner since it is less likely to introduce additional problems